immunisations/README.md

```diff
@@ -0,0 +1,162 @@
+| extension\[0] / url                                          | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-VaccinationProcedure-1` |
+| extension\[0] / valueExtension / url                         | fixed value = `https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid`                       |
+| extension\[0] / valueExtension / valueCodeableConcept / text | `ObservationStatement / code / displayname`                                                                   |
+| extension\[1] / url                                          | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-DateRecorded-1`         |
+| extension\[1] / valueString                                  | `ObservationStatement / effectiveTime / highValue`                                                            |
+| identifier / system                                          | fixed value = `https://PSSAdaptor/2167888433`                                                                 |
+| identifier / value \[@root]                                  | `ObservationStatement / id`                                                                                   |
+| status                                                       | fixed value = `COMPLETED`                                                                                     |
+| notGiven                                                     | fixed boolean value = `false`                                                                                 |
+| patient / reference                                          | reference to the mapped [patient](../patient/README.md)                                                       |
+| encounter / reference                                        | reference to the associated [encounter](../encounter/README.md)                                               |
+| date                                                         | `ObservationStatement / effectiveTime / highValue`                                                            |
+| primarySource                                                | fixed boolean value = `false`                                                                                 |
+| practitioner / actor / reference                             | `ObservationStatement / Participant / typeCode / agentRef / id / root`                                        |
+| note\[0] / text                                              | `ObservationStatement / pertiinentInformation / pertiantAnnotation / text` - Built from multiple                                     |
+| note\[1] / text[1]                                           | `ObservationStatement / effectiveTime` - Prints out the high value of effective time                          |
+
+## Unmapped Fields
+
+The following Immunization fields are not currently populated by the adaptor:
+
+- implicitRules

```

Comment:
I think there are some fields missing here. We should check it against this https://developer.nhs.uk/apis/gpconnect-1-6-0/accessrecord_structured_development_immunization.html